src/lib/abacus/index.ts

```diff
@@ -115,9 +118,9 @@ class AbacusStateManager {
     if (network) {
       this.stateManager.environmentId = network;
     }
-    this.stateManager.trade(null, null);
     this.stateManager.readyToConnect = true;
     this.setMarket(this.currentMarket ?? DEFAULT_MARKETID);
+    this.stateManager.trade(null, null);
```

Comment:
qq what does this do, why did we make field /type nullable?

Reply:
I think this was to match the existing iOS implementation (in hopes that it fixed some of the AsyncStateManagerV2 issues).